docs/spec/pagination/pagination.examples.centering-active-page-link.md

```diff
@@ -0,0 +1,30 @@
+8.10: Pagination Centering the active page link examples
+============================================
+
+ **Primary Actor**: User 
+ 
+ **Scope**: Ngx-bootstrap DEMO / BS version 3&4
+ 
+ **Goal**: Show user Centering the active page link functionality
+ 
+ Main success scenario:
+----------------------
+
+ 1. User opens "Pagination" demo page
+ 2. User clicks on "Centering the active page link" sub-menu
+ 3. By default maximum page numbers equal to 5 and active page 1.
+ 4. By default centering the active page link is enabled
+ 5. When user clicks on 5th page, then active page should 5 and should by center, maximum count of pages should 5
```

Comment:
When user click on 5th page, then active page - 5 and it centered, maximum count of pages - 5

Reply:
rewrite other